File: Atividades-SistemasDistribuidos/Unidade-II/3-Lista01Concorrente/Ex5SocketConcorrente/src/main/java/CategoriaNadador.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */


/**
 *
 * @author mathe
 */
public class CategoriaNadador {
    public String obterCategoria(int idade) {
        if (idade >= 5 && idade <= 7) {
            return "infantil A";
        } else if (idade >= 8 && idade <= 10) {
            return "infantil B";
        } else if (idade >= 11 && idade <= 13) {
            return "juvenil A";
        } else if (idade >= 14 && idade <= 17) {
            return "juvenil B";
        } else if (idade >= 18) {
            return "adulta";
        } else {
            return "sem categoria";
        }
    }
}

File: Atividades-SistemasDistribuidos/Unidade-II/2-Lista01PythonIterativo/Ex5/home05/CategoriaNadador.py
class CategoriaNadador:
    def obterCategoria(idade):
        if (idade >= 5 and idade <= 7):
            return "infantil A"
        elif (idade >= 8 and idade <= 10):
            return "infantil B"
        elif (idade >= 11 and idade <= 13):
            return "juvenil A"
        elif (idade >= 14 and idade <= 17):
            return "juvenil B"
        elif (idade >= 18):
            return "adulta"
        else:
            return "sem categoria"